tools/FakeItEasy.PrepareRelease/Program.cs

```diff
@@ -209,17 +209,17 @@ public static async Task UpdateRelease(this IGitHubClient gitHubClient, Release
         public static async Task CreateNextRelease(this IGitHubClient gitHubClient)
         {
             const string newReleaseBody = @"
-        ### Changed
+### Changed
 
-        ### New
-        * Issue Title (#12345)
+### New
+* Issue Title (#12345)
 
-        ### Fixed
+### Fixed
 
-        ### Additional Items
+### Additional Items
 
-        ### With special thanks for contributions to this release from:
-        * Real Name - @githubhandle";
+### With special thanks for contributions to this release from:
+* Real Name - @githubhandle";

```

Comment:
Not related to this PR, so I can move this commit to a separate PR if you like.

Reply:
Part of #1429, no? I've a fix at home, but hadn't time to push before coming to work. If it suits you, just drop it?

Reply:
Or if you like, take #1429 from me.